CDCgov/prime-simplereport
# Johanna/5681 reduce parameters add user mutation[Part2/3]

# FRONTEND PULL REQUEST

## Related Issue

- Implements second part of #5721

## Changes Proposed

- Modify the graphql operation addUser to pass all the user data as part of a user object.

## Additional Information

- An additional PR will be done to remove the deprecated fields in the mutation once this updates gets to prod

## Testing
Code is available in dev4 

- Add an admin to an org and verify that the addUser mutation is passing the user object
<img width="1004" alt="Screenshot 2023-05-02 at 12 46 29 PM" src="https://user-images.githubusercontent.com/103958711/235731457-ec4a7363-1574-495a-920b-db7679467231.png">
<img width="394" alt="Screenshot 2023-05-02 at 12 46 39 PM" src="https://user-images.githubusercontent.com/103958711/235731469-6682a499-9149-4fa6-a0d0-a4d12131e4ce.png">

- Verify that the user gets created in okta preview
